### DISCUSSION TITLE
Is UNO Platform supporting to deploy on IIS?

### DISCUSSION THREAD

**Discussion by tomchoondal** [unoplatform/uno#14637] [General]
I need to know more about the deployment of Uno Wasm project to IIS. I am able to run in IIS Express in debug mode. Tried couple of methods to deploy on IIS not working, Can any one help on this

**Comment by sasakrsmanovic** [comment]
Hello @tomchoondal 
In short, yes, Uno Platform applications can be deployed to IIS. You can see our documentation here https://platform.uno/docs/articles/how-to-host-a-webassembly-app.html#iis

Let us know if that is sufficient to get you going or we need to improve those docs.

**Reply by TopperDEL** [reply]
Sorry for bringing up this old topic - but for me deploying to IIS is a real pain now. I've used it before (I guess with Uno 3.1 or something) and that worked flawlessly. Now it doesn't.

I've created a new Uno-App from the latest template, ran it locally and then deployed it to IIS using "WebDeploy". I created a "Site" in IIS to deploy to - and now I'm stuck. The doc above mentiones to set the physical path to the distr-folder. BUT: before I have deployed the app, that folder does not exist (obviously). And if I change the path later, every further deploy creates a new folder-structure underneath that "old dist"-folder. This can't be right.

So I would love to have an updated documentation on how to correctly deploy WASM to IIS. This has been supereasy before, but I already got stuck three times trying to "test something quickly with Uno" and gave up because of this.
Thank you!

**Reply by TopperDEL** [reply]
I also have problems during deployment as VS states "MApp.Wasm.dll cannot be opened for writing - it is in use by another process".
Restarting the PC and VS does not help. I sometimes manage to get it working by running `dotnet build-server shutdown` on the console.

**Reply by TopperDEL** [reply]
The web.config on the server is empty after deployment. I think this might be the reason why no URL-rewrite is working

**Reply by TopperDEL** [reply]
This is really no fun. I changed the publish profile to "do AOT" and now it complains "project.assets.json does not contain a target for net8.0/win-x64".

**Reply by jeromelaban** [reply]
@TopperDEL can you open an issue with binlogs?

**Reply by TopperDEL** [reply]
I've created #16015
